operator/pkg/controller/karmada/planner.go

```diff
@@ -111,13 +113,24 @@ func (p *Planner) preRunJob() error {
 }
 
 func (p *Planner) runJobErr(err error) error {
+	var errs []error
+	errs = append(errs, err)
+
 	operatorv1alpha1.KarmadaFailed(p.karmada, operatorv1alpha1.Ready, err.Error())
-	return p.Client.Status().Update(context.TODO(), p.karmada)
+	errs = append(errs, p.Client.Status().Update(context.TODO(), p.karmada))
+
+	return utilerrors.NewAggregate(errs)
 }
 
 func (p *Planner) afterRunJob() error {
 	if p.action == InitAction {
+		// Update the karmada condition to Ready and set kubeconfig of karmad apiserver to karmada status.
```

Comment:
```suggestion
		// Update the condition to Ready and set kubeconfig of karmada-apiserver to status.
```

typo `karmad` --> karmad`a`.